### PR TITLE
[enterprise-4.13] OCPBUG 19873: Created Cri-O default capabilities section

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.13 

Issue: https://issues.redhat.com/browse/OCPBUGS-19873

Link to docs preview: https://78951--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html#create-crio-default-capabilities_post-install-machine-configuration-tasks

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by subhtk** [comment]
Related to https://github.com/openshift/openshift-docs/pull/68973

File: modules/create-crio-default-capabilities.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-crio-default-capabilities_{context}"]
+= Creating a drop-in file for CRI-O's default capabilities
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:8]:
```suggestion
= Creating a drop-in file for the default capabilities of CRI-O
```
Per IBM Style, avoid using the possessive with abbreviations or product names: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

File: modules/create-crio-default-capabilities.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+You can change some of the settings associated with the {product-title} CRI-O runtime for the nodes associated with a specific machine config pool (MCP). Using a controller custom resource (CR), you set the configuration values and add a label to match the MCP. The MCO then rebuilds the `crio.conf` and `default.conf` configuration files on the associated nodes with the updated values.
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:11]:
```suggestion
You can change some of the settings associated with the {product-title} CRI-O runtime for the nodes associated with a specific machine config pool (MCP). By using a controller custom resource (CR), you set the configuration values and add a label to match the MCP. The MCO then rebuilds the `crio.conf` and `default.conf` configuration files on the associated nodes with the updated values.
```
Per IBM Style word usage, avoid ambiguous instances of _using_ by replacing them with either _by using_ or _that uses_.

File: modules/create-crio-default-capabilities.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Machine Configs that once were a default in older {product-title} versions have not been removed by updating to newer {product-title} versions which do not include these Machine Configs as default. In order to ensure that clusters running on the same {product-title} version have the same default Machine Configs included.
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:13]: These two sentences are a bit hard to follow. Does this suggestion retain your intended meaning?
```suggestion
Earlier versions of {product-title} included specific machine configs by default. If you updated to a later version of {product-title}, those machine configs were retained to ensure that clusters running on the same {product-title} version have the same machine configs.
```

File: modules/create-crio-default-capabilities.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+You can create multiple `ContainerRuntimeConfig` CRs, as needed, with a limit of 10 per cluster. For the first `ContainerRuntimeConfig` CR, the MCO creates a machine config appended with `containerruntime`. With each subsequent CR, the controller creates a new `containerruntime` machine config with a numeric suffix. For example, if you have a `containerruntime` machine config with a `-2` suffix, the next `containerruntime` machine config is appended with `-3`.
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:15]:
```suggestion
You can create multiple `ContainerRuntimeConfig` CRs, as needed, with a limit of 10 per cluster. For the first `ContainerRuntimeConfig` CR, the MCO creates a machine config appended with `containerruntime`. With each subsequent CR, the controller creates a `containerruntime` machine config with a numeric suffix. For example, if you have a `containerruntime` machine config with a `-2` suffix, the next `containerruntime` machine config is appended with `-3`.
```
Minor edit to comply with IBM Style, which states that "create a new" is redundant and can be replaced by simply "create a".

File: modules/create-crio-default-capabilities.adoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+If you want to delete the machine configs, you should delete them in reverse order to avoid exceeding the limit. For example, you should delete the `containerruntime-3` machine config before deleting the `containerruntime-2` machine config.
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:17]:
```suggestion
If you want to delete the machine configs, delete them in reverse order to avoid exceeding the limit. For example, delete the `containerruntime-3` machine config before you delete the `containerruntime-2` machine config.
```

File: modules/create-crio-default-capabilities.adoc [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+[NOTE]
+====
+If you have a machine config with a `containerruntime-9` suffix, and you create another `ContainerRuntimeConfig` CR, a new machine config is not created, even if there are fewer than 10 `containerruntime` machine configs.
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:21]:
```suggestion
If you have a machine config with a `containerruntime-9` suffix and you create another `ContainerRuntimeConfig` CR, a new machine config is not created, even if there are fewer than 10 `containerruntime` machine configs.
```

File: modules/create-crio-default-capabilities.adoc [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+====
+
+.Example showing multiple `ContainerRuntimeConfig` CRs
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:24]:
```suggestion
.Example of multiple `ContainerRuntimeConfig` CRs
```

File: modules/create-crio-default-capabilities.adoc [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+.Example output
+[source,terminal]
+----
+NAME         AGE
+ctr-overlay  15m
+ctr-level    5m45s
+----
+
+.Example showing multiple containerruntime related system configs
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:38]:
```suggestion
.Example of multiple `containerruntime` related system configs
```

File: modules/create-crio-default-capabilities.adoc [hunk 9]
```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:6]: I see that this is listed as a procedure and the module title starts with a gerund, also indicating that this is a procedure. However, I don't see any steps. Can some of the paragraphs above the NOTE be restructured into steps?

File: modules/create-crio-default-capabilities.adoc [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+NAME         AGE
+ctr-overlay  15m
+ctr-level    5m45s
+----
+
+.Example showing multiple containerruntime related system configs
+[source,terminal]
+----
+$ cat /proc/1/status | grep Cap
+$ capsh --decode=<decode_CapBnd_value>
```
Comment (lahinson) [inline modules/create-crio-default-capabilities.adoc:42]: Does `<decode_CapBnd_value>` need a callout to specify what the user should replace that value with?

**Review by lahinson — COMMENTED** [review]
@subhtk I left a few comments for you. Most notably, I think we might need to either do some restructuring to add steps to this procedure, or reconsider whether it's actually a procedure.

File: modules/create-crio-default-capabilities.adoc (hunk 9 above)
Comment (subhtk) [reply modules/create-crio-default-capabilities.adoc:6]: Can I change it as CONCEPT? Initially when I started working it did have steps but those steps were no more relevant so it was restructured as the present module.

Comment (lahinson) [reply modules/create-crio-default-capabilities.adoc]: @subhtk Yes, I think it's acceptable to change it to a CONCEPT.

Comment (subhtk) [reply modules/create-crio-default-capabilities.adoc]: Great. Thanks
